### DISCUSSION TITLE
Should we stash the issues related to the frontend?

### DISCUSSION THREAD

**Discussion by nelimee** [tqec/tqec#323] [Q&A]
@QCHackers/tqec 

Since #321 is very close to being merged and we agreed to start a period of consolidation of the code base, I think this is the right moment to ask this question.

Currently, there are 26 issues opened with the `frontend` tag:

1. #18 
2. #19
3. #30 
4. #38 
5. #39 
6. #46 
7. #84 
8. #142 
9. #151 
10. #152 
11. #160 
12. #161 
13. #162 
14. #168 
15. #170 
16. #176 
17. #184 
18. #189 
19. #190 
20. #194 
21. #201 
22. #230 
23. #231 
24. #232 
25. #233 
26. #236 

Moreover, 5 issues do not have the tag but are frontend-related:

1. #214
2. #169 
3. #161 
4. #149 
5. #58 (the non-frontend-related part of that issue has been fixed)

Because the frontend-related work seems currently stale, I think we can close all these issues with the knowledge that, if someone eventually want to take back the frontend work where it was left, we will be able to come back to the above exhaustive list of issues and re-open them.

One uncertainty I have is on @giangiac version of the frontend. Should we leave the issues related to that open @giangiac or not?

**Comment by afowler** [comment]
+1 for closing out the front end issues, subject to Gian's input and
thoughts.

**Comment by giangiac** [comment]
Thanks Adrien to bring this to our attention. I have a couple of old PRs
that need to be merged. I'll handle this before our next call. Then we can
decide what to do with the other PRs which do not target the latest
front-end.

Gian

**Comment by nelimee** [comment]
@giangiac I will close all the issues listed in my initial message. Feel free to re-open any issue that is still relevant to your work. I let you manage the PRs.

**Comment by purva-thakre** [comment]
feel free to reopen if anyone wants to resume the conversation.
